packages/thirdweb/src/exports/utils.ts

```diff
@@ -169,3 +169,5 @@ export {
   getClaimParams,
   type GetClaimParamsOptions,
 } from "../utils/extensions/drops/get-claim-params.js";
+
+export type { NFTMetadata, NFTInput } from "../utils/nft/parseNft.js";
```

Comment:
add a changeset since ur updating the package (patch is fine here since its purely types)